test/parse.jl

```diff
@@ -321,3 +321,14 @@ end
     @test eltype([tryparse(Float64, s) for s in String[]]) == Union{Nothing, Float64}
     @test eltype([tryparse(Complex{Int}, s) for s in String[]]) == Union{Nothing, Complex{Int}}
 end
+
+@testset "isssue #29980" begin
+    @test parse(Bool, "1") == true
```

Comment:
We have `1 == true`, so it's better to use `===` here, as that ensures the type is correct as well.

Reply:
Do we?

```julia
julia> 1 === true
false
```

but I'm at school and don't have access to up-to-date Julia, so I don't know

Reply:
fixed